mtags/src/main/scala/scala/meta/internal/mtags/DefinitionAlternatives.scala

```diff
@@ -26,6 +29,27 @@ object DefinitionAlternatives {
       Some(sym.owner -> sym.value.desc)
   }
 
+  private def stripSyntheticPackageObjectFromObject(
```

Comment:
Do we need to validate this? Could there be an option where we have both object with the $package prefix and without?

And if we really need to we should have something like:

```
case class Alternative(private val symbol: Symbol, validation: (loc: SymbolLocation) => Boolean = true) {
  def symbol(loc: SymbolLocation) = {
   if (validation(loc)) Some(symbol)
   else None
  }
}
```

this way we make sure that Validation is not ignored at any point

Reply:
I think we can skip it... One would have to specifically create an object / package object called `something$package` (where I think that `$` is generally reserved for the compiler) and still this is only a fallback...

Reply:
Let's maybe skip it then.